### ISSUE TITLE
"Logged as" visible for all type of roles

### ISSUE THREAD

**Issue by mpingarron** [sebo-b/warp#4]
Hello,
Is there any possibility to see the "logged as" information for every user logged, not only for users with Manager role?
Great and amazing job done with WARP!
Thanks and regards!

**Comment by mpingarron** [comment]
Yes, that's right. When I logged as admin user, the "Logged as: username [login]]" is shown in the navbar at the right between "users" menu and warp logo. But when logged as a user, the block of the right (report, users and login) is not shown.

I have very limited knowledge of web programing, but I will try to do the best to help you and get it running. In case of get it, I will give you the code.

Thank you very much for your work!

**Comment by productiveme** [comment]
+1 
A function for the user to change their own password would be very helpful. Currently an admin has to either use a common password for all users (which negates having a password in the first place), or have to create different passwords for every user and communicate that manually. It makes life a lot harder for the admins.

**Comment by davidefu** [comment]
made here https://github.com/davidefu/warp/commit/1d461d22fd955b0055fb21cc484d4c44f4aaf51b
currently only showing the username
